test/e2e/declarative_test_fwk/model.py

```diff
@@ -54,10 +54,13 @@ class DeleteDict(ResourceDict):
 
 
 # fields for 'expect' field in a test step
-class ExpectDict(TypedDict, total=False):
+class ExpectDictK8S(TypedDict, total=False):
     spec: Dict
     status: Dict
 
+class ExpectDictBOTO3(TypedDict, total=False):
+    spec: Dict
+    status: Dict
```

Comment:
There is no such thing as a `spec` or `status` in the response of a boto3 resource. Instead, on line #74, we could just make it `expect_boto3: Dict`

Reply:
Got it